### ISSUE TITLE
Add container grouping

### ISSUE THREAD

**Issue by natecox** [natecox/corrator#10]
Group containers together in-config and run them by name.

```toml
[containers.ubuntu]
config = "here"

[containers."app name here"]
config = "here"

[containers.etc]
config = "here"

[groups.my-project-name]
containers = [ "ubuntu", "app name here", "etc" ]
```
 
```
corrator --group my-project-name
```

**Comment by natecox** [comment]
Alternate idea: add a `tags` property to containers instead and a `--filter` option to the cli.

**Comment by natecox** [comment]
Tagging is now included as of v0.2.1
